.github/workflows/ci.yml

```diff
@@ -31,15 +31,8 @@ jobs:
       - name: Applying clang-format for changed files
         shell: bash
```

Comment:
do we need this line? it doesn't look like that after the latest modification?

Reply:
No, we shouldn't need it anymore.  Good catch.